web/tests/lib/test_stream.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+let stream_id = 1;
+
+const get_stream_id = () => {
+    stream_id += 1 + Math.floor(Math.random() * 10);
+    return stream_id;
+};
+
+const get_random_string = () => Math.floor(Math.random() * 2 ** 54).toString(36);
```

Comment:
I think a better naming scheme would be of the format `stream-{random}`, so that when printing them, the types are more obvious.